### DISCUSSION TITLE
Hecate's Object Grouping

### DISCUSSION THREAD

**Discussion by kovitikus** [evennia/evennia#2535] [Community Contribs & Snippets]
Hecate is my MUD. You can find the article, as well as the source code for all of this logic, over on my GitHub. My project is both open-source and public domain.

https://github.com/kovitikus/hecate-mud/blob/master/docs/evennia/object_grouping.md

***

I was originally  going to manage the article here, but Discussions won't let me generate links for title sections, so I couldn't create a Table of Contents and that's a no from me dawg. GG.

**Comment by Griatch** [comment]
Hm, seems like you are right - github READMEs have link anchors just fine, but apparently it does not work here. Curious.

**Comment by Tharic99** [comment]
> Hecate is my MUD. You can find the article, as well as the source code for all of this logic, over on my GitHub. My project is both open-source and public domain.
> 
> https://github.com/kovitikus/hecate/blob/master/docs/evennia/object_grouping.md
> 
> I was originally going to manage the article here, but Discussions won't let me generate links for title sections, so I couldn't create a Table of Contents and that's a no from me dawg. GG.

Is your code stored anywhere else? Looks like the github is gone

**Reply by kovitikus** [reply]
@Tharic99

Yes, sorry. I had changed the name and visibility of the repository because I was starting a different project, but wanted to reuse the same name. Here's the changed name version, made public.
https://github.com/kovitikus/hecate-mud

And the markdown article that I wrote for this discussion topic.
https://github.com/kovitikus/hecate-mud/blob/master/docs/evennia/object_grouping.md

Here's the original class.
https://github.com/kovitikus/hecate-mud/blob/master/items/item_handler.py

Keep in mind that I barely knew what I was doing back then. If I were to do it all over again it'd be with proper classes and object instantiation and member variables and all of the stuff I didn't understand at the time. But, the logic itself is pretty solid. I hope that you will find it useful.
